tests/integration_tests/datasets/api_tests.py

```diff
@@ -1382,18 +1380,19 @@ def test_export_dataset_not_found(self):
         rv = self.get_assert_metric(uri, "export")
         assert rv.status_code == 404
 
+    @pytest.mark.usefixtures("create_datasets")
     def test_export_dataset_gamma(self):
         """
         Dataset API: Test export dataset has gamma
         """
-        birth_names_dataset = self.get_birth_names_dataset()
+        dataset = self.get_fixture_datasets()[0]
 
-        argument = [birth_names_dataset.id]
+        argument = [dataset.id]
         uri = f"api/v1/dataset/export/?q={prison.dumps(argument)}"
 
         self.login(username="gamma")
         rv = self.client.get(uri)
-        assert rv.status_code == 404
+        assert rv.status_code == 401
```

Comment:
Can you improve this test, granting `can_export` to gamma and testing that it can export after that?

I think you can do something like `security_manager.add_permission_role("Gamma", "dataset_can_export")` here to add the permission.